packages/reakit/src/Portal/Portal.tsx

```diff
@@ -8,14 +8,18 @@ export type PortalProps = {
   children: React.ReactNode;
 };
 
+const getBodyElement = () =>
+  typeof document !== "undefined" ? document.body : null;
```

Comment:
It's fine! I'd only write it as `function getBodyElement() { ...` to keep the convention.

Reply:
https://github.com/reakit/reakit/pull/514/commits/35fb93afae89aaba019f9a259d06ca124077905f